pkg/version/version.go

```diff
@@ -0,0 +1,61 @@
+package version
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	// commitFromGit is a constant representing the source version that
+	// generated this build. It should be set during build via -ldflags.
```

Comment:
that is, is -ldflags being set in our build process?

Reply:
Yes, https://github.com/openshift/image-registry/blob/c540fa0224d2028b9633a147c283fbf60d371ba9/hack/lib/constants.sh#L59-L63

Reply:
cool, thanks.